### PR TITLE
Add lineDashOffset Definition to types/openlayers

### PR DESCRIPTION
**Pull request by jansule**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://openlayers.org/en/v4.6.5/apidoc/ol.style.Stroke.html
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by osechet** [comment]
@jansule Thanks for your modification.  Could you add the corresponding unit tests, please?

**Comment by jansule** [comment]
@osechet Made the requested changes. Hope it's fine like this.

File: types/openlayers/index.d.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@
 //                 Yair Tawil <https://github.com/yairtawil>
 //                 Pierre Marchand <https://github.com/pierremarc>
 //                 Hauke Stieler <https://github.com/hauke96>
+//                 Jan Suleiman <https://github.com/jansule>
```
Comment (osechet) [inline types/openlayers/index.d.ts]: As explained in the DefinitelyTyped contributions guide, add yourself to the author list will cause you to be notified whenever someone makes a pull request. Is it really what you want?

File: types/openlayers/openlayers-tests.ts [hunk 2]
```diff
@@ -1111,6 +1112,35 @@ styleRegularShape = new ol.style.RegularShape({
...
+styleStroke.setLineCap('square');
+styleStroke.setLineJoin('bevel');
+styleStroke.setLineJoin('round');
+styleStroke.setLineJoin('miter');
+styleStroke.setLineDash([10, 5]);
+styleStroke.setLineDashOffset(10);
+styleStroke.setMiterLimit(20);
+styleStroke.setWidth(5);
+
+styleStroke.getColor();
```
Comment (osechet) [inline types/openlayers/openlayers-tests.ts]: For the getters, you should affect the returned value to typed variables.

**Review by osechet — CHANGES_REQUESTED** [review]

File: types/openlayers/index.d.ts (hunk 1 above)
Comment (jansule) [reply types/openlayers/index.d.ts]: Good point there.

File: types/openlayers/openlayers-tests.ts (hunk 2 above)
Comment (jansule) [reply types/openlayers/openlayers-tests.ts]: Did so with latest commit.

**Review by osechet — APPROVED** [review]

**Pull request merged**